src/display/annotation_layer.js

```diff
@@ -206,6 +206,7 @@ class AnnotationElement {
 
     const container = document.createElement("section");
     container.setAttribute("data-annotation-id", data.id);
+    container.setAttribute("tabindex", DEFAULT_TAB_INDEX);
```

Comment:
Looking at existing code, could we use `container.tabIndex = DEFAULT_TAB_INDEX;` instead?
Furthermore, it seems strange to set this unconditionally and then remove it for certain Annotation-types. Can't we simply do the following instead?
```suggestion
    if (!(this instance of WidgetAnnotationElement)) {
      container.tabIndex = DEFAULT_TAB_INDEX;
    }
```